tf/getter.go

```
@@ -174,6 +197,57 @@ func (tfrGetter *RegistryGetter) Get(dstPath string, srcURL *url.URL) error {
+	modulePath = strings.TrimPrefix(modulePath, "/")
+	moduleVersionsPath := fmt.Sprintf("%s/%s/versions", moduleRegistryBasePath, modulePath)
+	moduleVersionsURL, err := url.Parse(moduleVersionsPath)
+	if err != nil {
+		return "", errors.New(err)
+	}
+	body, _, err := httpGETAndGetResponse(ctx, logger, *moduleVersionsURL)
+	if err != nil {
+		return "", errors.New(ModuleVersionsErr{moduleName: modulePath})
+	}
+	var responseJSON Modules
+	json.Unmarshal(body, &responseJSON)
```

golangci-lint (1.62.2)

Error return value of `json.Unmarshal` is not checked

(errcheck)